web/ui/module/codemirror-promql/LICENSE

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
I agree.

Reply:
OK for me

Reply:
yup better to be  aligned with the license. Let's see if everybody is agree with that.

Reply:
@dsmith3197 any chance to have an answer from you ? It will be cool to know if it's fine for you as well to change the license of codemirror-promql from MIT to Apache v2 :).

Reply:
I'm fine with that. I apologize for the delayed response.

Reply:
cool thank you very much for your answer dsmith3197